x/payment/keeper/stream_record.go

```diff
@@ -123,7 +124,12 @@ func (k Keeper) UpdateFrozenStreamRecord(ctx sdk.Context, streamRecord *types.St
 		streamRecord.LockBalance = streamRecord.LockBalance.Add(change.LockBalanceChange)
 		streamRecord.StaticBalance = streamRecord.StaticBalance.Sub(change.LockBalanceChange)
 		if streamRecord.LockBalance.IsNegative() {
-			return fmt.Errorf("lock balance can not become negative, current: %s", streamRecord.LockBalance)
+			if ctx.IsUpgraded(upgradetypes.Pawnee) {
+				streamRecord.StaticBalance = streamRecord.StaticBalance.Add(streamRecord.LockBalance)
```

Comment:
what if staticbalance become negative? is it possible?

Reply:
For the stream record is frozen, it will be negative. 
When deposit, the user need to cover the fund.

<img width="1060" alt="image" src="https://github.com/bnb-chain/greenfield/assets/61674316/3fa650c6-c533-4253-93ce-30cc9304fbd6">